augur/application/schema/alembic/versions/12_traffic_additions.py

```diff
@@ -0,0 +1,71 @@
+"""traffic additions
+
+Revision ID: 12
+Revises: 11
+Create Date: 2022-12-30 19:23:17.997570
+
+"""
+from alembic.autogenerate import renderers
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+from sqlalchemy.schema import Sequence 
+
+# revision identifiers, used by Alembic.
+revision = '12'
+down_revision = '11'
+branch_labels = None
+depends_on = None
+
+traffic_sequence = Sequence('repo_clones_data_id_seq', schema='augur_data')
+
+# Current Error
+    #   File "/home/sean/github/berkeley/augur/application/schema/alembic/versions/12_traffic_additions.py", line 38, in add_repo_clone_data_table_1
+    #     op.execute(schema.CreateSequence(traffic_sequence))
+    # NameError: name 'schema' is not defined
+
+def upgrade():
+
+    add_repo_clone_data_table_1()
+
+def downgrade():
+
+    upgrade = False
+
+    add_repo_clone_data_table_1(upgrade)
+    
```

Comment:
There doesn't need to be function defined for this. That upgrades when upgrade is True and downgrades when it is False. Instead the upgrade logic should up put into the `upgrade` function and the `downgrade` logic should be put into the `downgrade` function.

Reply:
OK. It works as it is now. I'm not against a change, but this is expensive time wise to test and validate, and I've done that for what's here.